mangaki/mangaki/tests/test_anidb.py

```diff
@@ -70,3 +74,16 @@ def test_anidb_get(self):
         self.assertCountEqual(author_names, ['Umino Chika'])
         self.assertCountEqual(composer_names, ['Hashimoto Yukari'])
         self.assertCountEqual(director_names, ['Shinbou Akiyuki', 'Okada Kenjirou'])
+
+        self.assertCountEqual(tag_titles, ['seinen', 'high school', 'dynamic', 'target audience',
+                                           'themes', 'original work', 'setting', 'elements',
+                                           'time', 'place', 'present', 'Earth', 'Japan',
+                                           'Tokyo', 'board games', 'manga', 'Asia', 'comedy',
+                                           'anthropomorphism', 'school life', 'sports',
+                                           'shougi', 'aim for the top', 'funny expressions',
+                                           'male protagonist', 'hard work and guts',
+                                           'dysfunctional family', 'following one`s dream',
+                                           'cast', 'family life'])
+
+        self.assertCountEqual(retrieve_tags["kept_tags"], tag_titles)
+        self.assertCountEqual(retrieve_tags["deleted_tags"], [])
```

Comment:
Here, you can skip this test and use `self.assertEqual(len(retrieve_tags["deleted_tags"]), 0)` or testing using boolean value (a list is empty if and only if `bool(instance) == False` because AFAIK `bool(instance)` uses `instance.__bool__` (magic method [1]) which uses `len(instance) > 0` or something like that.).

[1] : Magic methods in Python are the "equivalent" (it's not really operator overloading, and Python has a lot more magic methods than C++) to `bool operator > (MyClass const& lhs, MyClass const& rhs)` in C++ or anything related to operator overloading in other languages.